sources/osg/Matrix.js

```diff
@@ -65,6 +65,15 @@ var Matrix = {
         return matrix;
     },
 
+    equal: function ( a, b ) {
+        if ( a === b ) return true;
+        if ( a === undefined || b === undefined ) return false;
```

Comment:
better to use if ( !a ) instead of undefined. It will be valid for undefined and null